Externals.cfg

```diff
@@ -1,6 +1,6 @@
 [regional_workflow]
 protocol = git
-repo_url = https://github.com/ufs-community/regional_workflow
+repo_url = https://github.com/NOAA-EPIC/regional_workflow
```

Comment:
@mark-a-potts, why was this url changed to NOAA-EPIC?

Reply:
Ugh. It should not have been. I will put in a PR to get that fixed.

Reply:
Just put in a fix. It is PR #260